Connector service moved to Istio 1.1 (#4200)
kyma-project/kyma: components/connector-service/README.md

- **tokenLength** - Length of registration tokens. The default value is `64`.
- **appTokenExpirationMinutes** - Time after which tokens for Applications expire and are no longer valid. The default value is `5` minutes.
- **runtimeTokenExpirationMinutes** - Time after which tokens for Runtimes expire and are no longer valid. The default value is `10` minutes.

> - **caSecretName** - Namespace and Name of the secret, in Namespace/Name format, which contains the certificate and key used for signing client certificates. The default value is `kyma-integration/nginx-auth-ca`.

Reviewer: ```suggestion
- **caSecretName** - Namespace and the name of the Secret which stores the certificate and key used to sign client certificates. Requires the `Namespace/secret name` format. The default value is `kyma-integration/nginx-auth-ca`.
```

> - **rootCACertificateSecretName** - Namespace and Name of the secret, in Namespace/Name format, which contains root CA (Certificate Authority) Certificate in case certificates are singed by intermediate CA.

Reviewer: ```suggestion
- **rootCACertificateSecretName** - Namespace and the name of the Secret which stores the root CA (Certificate Authority) Certificate in case the certificates are signed by the intermediate CA. Requires the `Namespace/secret name` format. 
```

- **runtimeCertificateValidityTime** - Time until which the certificates that the service issues for Runtimes are valid. The default value is 90 days.
- **central** - Determines whether the Connector Service works in the central mode.
- **revocationConfigMapName** - Name of the ConfigMap containing the revoked certificates list.

> - **lookupEnabled** - Flag that determines whether connector should make a call to get gateway endpoint. It is set to `False` by default.

Reviewer: ```suggestion
- **lookupEnabled** - Determines if the Connector should make a call to get the gateway endpoint. The default value is`False`.
```

> - **lookupConfigMapPath** - Path in the pod where Config Map for cluster lookup will be stored. Default value is `/etc/config/config.json`. It is used only when `lookupEnabled` is set to `True`

Reviewer: ```suggestion
- **lookupConfigMapPath** - Path in the Pod where Config Map for cluster lookup is stored. The default value is `/etc/config/config.json`. Used only when **lookupEnabled** is set to `True`
```
